fitbenchmarking/results_processing/visual_pages.py

```diff
@@ -171,11 +107,16 @@ def get_figure_paths(support_pages_dir, problem_name, count):
     :rtype : tuple(str, str)
     """
 
-    figures_dir = os.path.join(support_pages_dir, "figures")
-    figure_fits = os.path.join(figures_dir, "Fit_for_" + problem_name +
-                               "_" + str(count) + ".png")
-    figure_strt = os.path.join(figures_dir, "start_for_" + problem_name +
-                               "_" + str(count) + ".png")
+    if result.figure_link == '' and result.start_figure_link == '':
+        message = 'Re-run with make_plots set to yes in the ini file to' \
```

Comment:
With make_plots off the output looks like this:
![plotting](https://user-images.githubusercontent.com/15614849/72439865-a5403e00-379f-11ea-81df-fba530f09533.png)
Is it possible to remove the directory name before the message?